doc/users/prev_whats_new/whats_new_3.6.0.rst

```diff
@@ -0,0 +1,564 @@
+==============================
+What's new in Matplotlib 3.6.0
+==============================
+
+For a list of all of the issues and pull requests since the last revision, see
+the :ref:`github-stats`.
+
+.. contents:: Table of Contents
+   :depth: 4
+
+.. toctree::
+   :maxdepth: 4
+
+Figure and Axes creation / management
+=====================================
+
+``subplots``, ``subplot_mosaic`` accept *height_ratios* and *width_ratios* arguments
+------------------------------------------------------------------------------------
+
+The relative width and height of columns and rows in `~.Figure.subplots` and
+`~.Figure.subplot_mosaic` can be controlled by passing *height_ratios* and
+*width_ratios* keyword arguments to the methods. Previously, this required
+passing the ratios in *gridspec_kws* arguments.
+
+New ``layout_engine`` module
+----------------------------
+
+Matplotlib ships with ``tight_layout`` and ``constrained_layout`` layout
+engines.  A new `.layout_engine` module is provided to allow downstream
+libraries to write their own layout engines and `~.figure.Figure` objects can
+now take a `.LayoutEngine` subclass as an argument to the *layout* parameter.
+
```

Comment:
```suggestion
```suggestion

Compressed layout for fixed-aspect ratio axes
----------------------------------------------

Simple arrangements of axes with fixed aspect ratios can now be packed together with 
``fig, axs = plt.subplots(2, 3, layout='compressed')``.  With ``layout='tight'` or `'constrained'`, axes
with a fixed aspect ratio can leave large gaps between the axes.  Using ``layout='compressed'`` reduces 
the space between the axes, and adds the extra space to the outer margins.  See `compressed_layout_`.
```

You may need to check the rst link.  The anchor is `_compressed_layout` in `tutorials/intermediate/constrainedlayout_guide.py`.